tests/behat/features/operate-subsites.feature

```diff
@@ -25,6 +25,7 @@ Feature: Create and select a subsite
 
     # Add a file to the main site
     When I go to "admin/assets"
+    And I select "Main site" from "SubsitesSelect"
```

Comment:
```suggestion
```
That's being done after "add folder" two lines down.

Reply:
Removed